### PR TITLE
Update cookie version

### PR DESCRIPTION
**Pull request by DrewScoggins**

### CODE REVIEW

File: src/OrchardCore.Themes/TheAgencyTheme/Assets/package.json [hunk 1]
```diff
@@ -40,7 +40,8 @@
     },
     "devDependencies": {
         "autoprefixer": "10.4.4",
-        "browser-sync": "2.27.9",
+        "browser-sync": "2.29.3",
```
Comment (LoopedBard3) [inline src/OrchardCore.Themes/TheAgencyTheme/Assets/package.json]: Is there any particular reason browser-synch is not updated in the two other themes?

**Review by LoopedBard3 — APPROVED** [review]
Left a comment but looks good otherwise.

File: src/OrchardCore.Themes/TheAgencyTheme/Assets/package.json (hunk 1 above)
Comment (DrewScoggins) [reply src/OrchardCore.Themes/TheAgencyTheme/Assets/package.json]: I need to put it back. This was an earlier fix.

**Comment by DrewScoggins** [comment]
I also updated all of the lock jsons as well.

**Pull request merged**